src/app/debug/table/table.component.ts

```diff
@@ -418,30 +463,32 @@ export class TableComponent implements OnInit, OnDestroy {
     let selectedReports: string[] = this.tableSettings.reportMetadata
       .filter((report) => report.checked)
       .map((report) => report.storageId);
-    this.httpService.getReports(selectedReports, this.viewSettings.currentView.storageName).subscribe({
-      next: (data) => {
-        const leftObject = data[selectedReports[0]];
-        const originalReport = leftObject.report;
-        originalReport.xml = leftObject.xml;
-
-        const rightObject = data[selectedReports[1]];
-        const runResultReport = rightObject.report;
-        runResultReport.xml = rightObject.xml;
-
-        const id = this.helperService.createCompareTabId(originalReport, runResultReport);
-        compareReports = {
-          id: id,
-          originalReport: originalReport,
-          runResultReport: runResultReport,
-          viewName: this.viewSettings.currentView.name,
-          nodeLinkStrategy: this.viewSettings.currentView.nodeLinkStrategy,
-        };
-      },
+    if (this.viewSettings.currentView?.storageName) {
+      this.httpService.getReports(selectedReports, this.viewSettings.currentView.storageName).subscribe({
+        next: (data) => {
+          const leftObject = data[selectedReports[0]];
+          const originalReport = leftObject.report;
+          originalReport.xml = leftObject.xml;
+
+          const rightObject = data[selectedReports[1]];
+          const runResultReport = rightObject.report;
+          runResultReport.xml = rightObject.xml;
+
+          const id = this.helperService.createCompareTabId(originalReport, runResultReport);
+          compareReports = {
+            id: id,
+            originalReport: originalReport,
+            runResultReport: runResultReport,
+            viewName: this.viewSettings.currentView?.name ?? '',
+            nodeLinkStrategy: this.viewSettings.currentView?.nodeLinkStrategy,
+          };
+        },
 
-      complete: () => {
-        this.tabService.openNewCompareTab(compareReports);
-      },
-    });
+        complete: () => {
+          this.tabService.openNewCompareTab(compareReports);
+        },
```

Comment:
This complete should not be here, if an error occurs in this http request it will still try to open a new compare tab, we've talking about this last friday.

Reply:
Yeah we did, I'm not sure why that's there again but I fixed it.